app/src/main/java/com/dvm/appd/oasis/dbg/MainActivity.kt

```diff
@@ -285,6 +297,9 @@ class MainActivity : AppCompatActivity(), NetworkChangeNotifier {
             Log.e("Main Activity", "Failed to recive token")
             setupNotificationChannel()
         }
+    }catch (e:Exception){
+
+    }
```

Comment:
If you want to leave a block blank, don't add extra lines.
`} catch(e: Exception) { }`
Try following this.

Reply:
Indentation corrected in the latest commit.